### DISCUSSION TITLE
[Open-webui] User account can not access local model After granting permission , but admin account can access normally

### DISCUSSION THREAD

**Discussion by Jacky-tao** [open-webui/open-webui#7927] [General]
Hi all
i meet a bug, deploy open-webui, admin account can access local model list and use, but new create a user account, then granting permission under administer control panel, find user account can not access local model,but admin account can access.

Please help provide suggestions and solutions，thank you for your understanding and support!

## Installation Method
Install Open-webui by docker or 1 panel APP, 

## Environment

- **Open WebUI Version:*v0.4.8* 
- **Ollama (if applicable):*v0.5.1"

- **Operating System:*Ubuntu 20.04]* 
- **Browser (if applicable):*Firefox 132.0.1* 

## Expected Behavior:
Both user and admin account can access local model list then use.

## Actual Behavior:
User account can not access local model After granting permission , but admin account can access

## Description
User account can not access local model After granting permission , but admin account can access
when issue occurs, try reinstall OS then deploy, the bug still exist.

## Reproduction Details

**Steps to Reproduce:**
1.Install Ubuntu22.04 OS
2.Install 1 panel
3.install ollama in 1 panel APP
4.download any local model(eg: llava3)
5.Install open-webui in 1 panel app
6.create admin account and login, can access and use local model list.
7.create user account then granting permission under admin panel(setting can refer to attached picture)
8.log out admin account then log in user account, find user account can not see local model list, issue occurs.

![setting](https://github.com/user-attachments/assets/6185f88e-99c0-4dd6-ae80-7b8dcb619f56)

**Comment by mahenning** [comment]
The permission "Models access" only enable the user (group) to create new model cards in the "Workspace" (left menu directly under "New Chat"). To let users see a model, you have to set the model visibility either to "public", or add the group to a private model. To access the model card, see [the screenshot in my other answer](https://github.com/open-webui/open-webui/issues/7931#issuecomment-2551333546). Models created in the workspace can be found in "Workspace -> Models" and have the same option on their model card.
(I find the naming a bit unfavourable, as I also first thought this enables access to all models)

**Reply by Jacky-tao** [reply]
thanks, refer to your methods, user account can see and use local models.

**Comment by Jacky-tao** [comment]
thanks support

**Comment by Jacky-tao** [comment]
close it
